lib/Parser/rterrors.h

```diff
@@ -363,6 +363,7 @@ RT_ERROR_MSG(JSERR_InvalidOperationOnTypedArray, 5664, "", "The operation is not
 RT_ERROR_MSG(JSERR_CannotSuspendBuffer, 5665, "", "Current agent cannot be suspended", kjstRangeError, 0)
 RT_ERROR_MSG(JSERR_CantDeleteNonConfigProp, 5666, "Cannot delete non-configurable property '%s'", "Cannot delete non-configurable property", kjstTypeError, 0)
 RT_ERROR_MSG(JSERR_CantRedefineProp, 5667, "Cannot redefine property '%s'", "Cannot redefine property", kjstTypeError, 0)
+RT_ERROR_MSG(JSERR_FunctionArgument_NeedArrayLike, 5668, "%s: argument is not an array-like object", "Array-like object expected", kjstTypeError, 0)
```

Comment:
Arrays are array-like objects. The spec uses the term ArrayLike in this context.